### DISCUSSION TITLE
Support for MPTCP

### DISCUSSION THREAD

**Discussion by miniman13** [SmoothWAN/SmoothWAN#140] [Ideas and feature requests]
Would be amazing if this project could also implement https://www.openmptcprouter.com/ Open MPTCP, so we can select a FOSS option that supports real Aggregation between 2 connections.

**Comment by TalalMash** [comment]
May add UBOND (fork of MLVPN) next release for FOSS aggregation, designed for **good** connections only, no reliability features like Speedify.

MPTCP is designed to be run on the client not tunnels/proxies and is application specific, not to mention ISP firewall filtering issues (TCP options). Also requires a lossless connection from each WAN especially for non-TCP traffic tunnel, unless it's in backup mode (no aggregation).
